content/docs/command-reference/freeze.md

```diff
@@ -1,32 +1,32 @@
 # freeze
 
-Freeze a [stage](/doc/command-reference/run). Use `dvc unfreeze` to unfreeze the
-stage.
+Fix the status of one or more [stages](/doc/command-reference/run) to _not
+changed_ until `dvc unfreeze` is used on it.
 
 ## Synopsis
 
 ```usage
 usage: dvc freeze [-h] [-q | -v] targets [targets ...]
 
 positional arguments:
-  targets        stages to freeze.
+  targets        Stages or .dvc files to freeze
```

Comment:
@pared please notice these updates which should probably be applied to the help output (dvc freeze -h) in the core repo.

Reply:
Actually. Are .dvc files still considered stages? Not sure we want to mention them separately but probably yes: "Stages" refers to their names (in dvc.yaml), .dvc file are paths to such files.

Reply:
I see this is already applied in iterative/dvc/pull/4102. Resolving